.github/workflows/generator_generic_slsa3.yml

```diff
@@ -115,7 +115,7 @@ jobs:
       - name: Detect the generator ref
         id: detect
         continue-on-error: true
-        uses: slsa-framework/slsa-github-generator/.github/actions/detect-workflow@main
+        uses: slsa-framework/slsa-github-generator/.github/actions/detect-workflow-js@main
```

Comment:
we could as well rename the js action as `detect-workflow`. Maybe a follow-up PR.

Reply:
Yes. I was going to do it on this PR but it just made it unreadable so I'll do it in a follow up.